artemis-journal/src/main/java/org/apache/activemq/artemis/journal/ActiveMQJournalBundle.java

```diff
@@ -39,7 +39,7 @@
    @Message(id = 149001, value = "Journal data belong to a different version")
    ActiveMQIOErrorException journalDifferentVersion();
 
-   @Message(id = 149002, value = "Journal files version mismatch. You should export the data from the previous version and import it as explained on the user''s manual")
```

Comment:
AFAIK this is not a typo.. it's how the logger works.. otherwise the ' would be translated as something else?